### DISCUSSION TITLE
네이버 지도가 안뜨고 개발자 모드에 에러가 뜹니다 (사진있어요!)

### DISCUSSION THREAD

**Discussion by yyvfff** [changdoc/naver-vaccine-macro#266] [Q&A]
이렇게 뜨는데 뭐가 문제일까요? 혼자 해결이 어렵습니다ㅜㅜ

매크로 돌리다가 창 닫고 다시 켰더니 이렇게 안됩니다

도와주시면 감사하겠습니다!

**Comment by changdoc** [comment]
아직까지 저는 재현이 된적 없는데 여러분들 께서 얘기를 주시고 계신 오류네요.

주소 창을 보니 https://m.place.naver.com/rest/vaccine?vaccine

이렇게 되어 있는데요. 

https://m.place.naver.com/rest/vaccine?vaccineFilter=unused

내일은 이 주소로 해보시겠어요?

**Reply by changdoc** [reply]
https://github.com/changdoc/naver-vaccine-macro/issues/271

코멘트에 적긴했는데 네이버 웨일로 한번 해보시겠어요?

**Reply by changdoc** [reply]
음.. 오래 돌리니 웨일도 동일하게 CORS 에러가 발생하네요..
